package.json

```diff
@@ -2,12 +2,12 @@
   "scripts": {
     "build": "esy",
     "build:watch": "esy && esy build:watch",
-    "bundle": "webpack --mode production --entry ./_build/default/src/ReactApp.js",
-    "serve": "webpack serve --open --mode development --entry ./_build/default/src/ReactApp.js"
+    "bundle": "webpack --mode production --entry ./_build/default/src/output/src/ReactApp.js",
+    "serve": "webpack serve --open --mode development --entry ./_build/default/src/output/src/ReactApp.js"
   },
   "dependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^16.8.1",
```

Comment:
why was react downgraded?

Reply:
I think this change comes from the opam template. In https://github.com/melange-re/melange-opam-template/pull/7 I replaced `rescript-react` with `reason-react` because I think if ReScript compat is not guaranteed by Melange in the future, and we are going to be recommending `reason-react`, that's what we should use on the templates.

We can upgrade again to v18 once `reason-react` supports it cc @davesnx 

Wdyt?

Reply:
If we go with `reason-react`, the readme should be updated. I just did the same in the opam template: https://github.com/melange-re/melange-opam-template/commit/c6e8f81ce2e7e012413965e14af46726a4cd20c5

Reply:
> If we go with `reason-react`, the readme should be updated. I just did the same in the opam template: [melange-re/melange-opam-template@c6e8f81](https://github.com/melange-re/melange-opam-template/commit/c6e8f81ce2e7e012413965e14af46726a4cd20c5)

Good call, readme updated.